jar/backup.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package jar
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+func BackupFiles(jar string) {
+	pathdst := filepath.Join(".", "backup")
+	if _, err := os.Stat(pathdst); os.IsNotExist(err) {
+		err := os.Mkdir(pathdst, os.ModePerm)
+		if err != nil {
+			fmt.Errorf("create backup %v", err)
+		}
+	}
+
+	jarSplit := strings.Split(jar, "/")
```

Comment:
Note that you need to use the path/filepath package when parsing or building paths. This code as written won't work on Windows.

https://pkg.go.dev/path/filepath#Split